posts/2021-10-29-kube-secrets-210011.adoc

```diff
@@ -161,7 +161,7 @@ dependencies {
 }
 ----
 
-Install the user feature using the link:https://openliberty.io/docs/latest/reference/command/featureUtility-installFeature.html[`featureUtility`] command-line tool, Liberty Maven or Gradle plugin. Provide the maven coordinate of the custom made `features-bom` file.
+Install the user feature using the link:https://openliberty.io/docs/latest/reference/command/featureUtility-installFeature.html[`featureUtility`] command-line tool, Liberty Maven or Gradle plugin. Provide the Maven coordinate of the custom made `features-bom` file.
```

Comment:
custom-made

Provide the Maven coordinate of the custom made features-bom file.
--->
Provide the Maven coordinates of the features-bom file that you created for the feature.